lib/config/theme/asset_icons.dart

```diff
@@ -16,6 +16,18 @@ class AssetIcons {
   static const knaagdieren = '$_path/animal-species/knaagdieren.svg';
```

Comment:
Change 'knaagdieren' and 'roofdieren' into 'rodents' and 'predators' for international students